src/components/Slider.js

```diff
@@ -61,16 +60,16 @@ class Slider extends React.Component {
   }
 
   componentWillUnmount() {
-    typeof window !== 'undefined' && // eslint-disable-line no-unused-expressions
-      window.removeEventListener('resize', this.updateResponsiveView);
+    typeof window !== 'undefined' // eslint-disable-line no-unused-expressions
+      && window.removeEventListener('resize', this.updateResponsiveView);
   }
 
   updateResponsiveView() {
     const { children, hideArrowsOnNoSlides } = this.props;
     let { responsive } = this.props;
     const numberOfChildren = children ? children.length || 1 : 0;
     if (responsive) {
-      responsive = sortBy(responsive, 'breakPoint');
+      responsive = responsive.map(obj => Object.assign({}, obj)).sort((key => (a, b) => (a[key] > b[key]) ? 1 : ((b[key] > a[key]) ? -1 : 0))('breakPoint'));
```

Comment:
Can you please break it into multiple lines.

Reply:
Hi broke on 2 lines, tell me if need more changes.